### PR TITLE
Add registration deadline, update agenda and abstract book

### PR DESCRIPTION
**Pull request by jennaswa**

### CODE REVIEW

File: public/data/mdaUGM2024-abstracts.pdf
Comment (orbeckst) [inline public/data/mdaUGM2024-abstracts.pdf:1]: In abstract **Kinisi: Bayesian Analysis of Mass Transport from Molecular Dynamics Simulations**

```
Self-diffusion coefficients, $\D$, are routinely estimated from molecular dynamics simulations by fitting a linear model to the observed mean-squared displacements (MSDs) of mobile species. MSDs derived from simulation suffer from statistical noise, which introduces uncertainty in the resulting estimate of $\D$. An optimal scheme for estimating $\D$ will minimise this uncertainty, i.e., will have high statistical efficiency, and will give an accurate estimate of the uncertainty itself. We present a scheme for estimating $\D$ from a single simulation trajectory with high statistical efficiency and accurately estimating the uncertainty in the predicted value. The statistical distribution of MSDs observable from a given simulation is modelled as a multivariate normal distribution using an eestimated covariance matrix, which we parameterise from the available simulation data. We then perform Bayesian regression to sample the distribution of linear models that are compatible with this model multivariate normal distribution, to obtain a statistically efficient estimate of $D^*$ and an accurate estimate of the associated statistical uncertainty. This methodology is implemented in an open-source Python packaged called kinisi.
```

replace the `$...$` characters with LaTeX formatted characters
* `$\D$`: $D$
* `$D^*$`: $D^*$

Comment (orbeckst) [reply public/data/mdaUGM2024-abstracts.pdf:1]: In abstract **Choline Based Plastic Crystals as Barocaloric Materials: Insights from Ab Initio Molecular Dynamics** format superscripts and subscripts:

```
(C_5 H_14 NO)_2 MX_4 where M= Co, Zn, Cu, and X= Cl, Br, I, as promising barocaloric materials due to a large entropy change (~100 JK^(-1) kg^(-1)) associated with the symmetry-breaking disorder-order phase transition. Using ab initio molecular dynamics as implemented in CP2K, we provide insights into the structural dynamics and hydrogen bonding between the choline and CoCl_4 that characterise the two phases. From our calculation of vibrational density of states, we estimate the entropy change across the phase transition. The choline and CoCl_4 
```

* `(C_5 H_14 NO)_2 MX_4`: $(\text{C}\_{5}\text{H}\_{14}\text{NO})\_{2}\text{MX}\_4$
* `(~100 JK^(-1) kg^(-1))`: (~100 JK<sup>-1</sup>kg<sup>-1</sup>)
* `CoCl_4`: CoCl<sub>4</sub>

**Review by orbeckst — APPROVED** [review]
Changes look good.

I have some formatting fixes for the abstracts booklet (which otherwise looks awesome!). These fixes can be made later so I am not blocking over them.

**Review by micaela-matta — APPROVED** [review]

File: public/data/mdaUGM2024-abstracts.pdf
Comment (jennaswa) [reply public/data/mdaUGM2024-abstracts.pdf:1]: Updated

**Pull request merged**